classes/core-image-pelux.bbclass

```diff
@@ -25,12 +25,13 @@ IMAGE_INSTALL += "\
     node-state-manager \
 "
 
-# OTA mechanism
-IMAGE_INSTALL_append_rpi += "\
+
+IMAGE_INSTALL += " \
     swupdate \
+    grub-efi \
```

Comment:
Does this mean that grub would also be installed on rpi? Was that the intention?

Reply:
no, this was just to get things work quickly to test how it works with intel. This needs to be fixed.